ghostscript/fix-sprintf.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/cups/gdevcups.c b/cups/gdevcups.c
```

Comment:
Could we add a comment at the top please, of where the original patch comes from?

Reply:
of course 🙉